src/main/clm_instMod.F90

```diff
@@ -440,10 +438,8 @@ subroutine clm_instInit(bounds)
 
     ! use_fates_spitfire is assigned an integer value in the namelist
     ! see bld/namelist_files/namelist_definition_clm4_5.xml for details
-    if (use_fates_spitfire > 1) then
-       call fates_fire_data_inst%InitAccBuffer(bounds)
-    else
-       call fates_fire_no_data_inst%InitAccBuffer(bounds)
+    if (use_fates_spitfire > 0) then
```

Comment:
I actually suggest that you keep the accumulation interfaces in both the FATESFireNoData and FATESfireData classes, so that you don't have to have this if statement. But, if you keep the if statement you don't have to have empty methods for the FATESFireNoData class. So it can be done either way...

Reply:
Ok, I'm keeping the acc interfaces in both (empty in the NoData case) and removing the if statement.